collabwrapper/test/activity/activity.info

```diff
@@ -0,0 +1,9 @@
+[Activity]
+name = CollabWrapperTest
+activity_version = 1
+bundle_id = org.sugarlabs.CollabWrapperTest
+exec = sugar-activity3 activity.CollabWrapperTestActivity
+icon = activity-collabwrappertest
+license = GPLv2+
+repository = https://github.com/sugarlabs/collabwrapper
+summary = Test CollabWrapper
```

Comment:
Lol, This too. These are actually test files to check collaboration

Reply:
Done in commit `766b774`